app/server/src/main/scala/org/bitcoins/server/BitcoindRpcBackendUtil.scala

```diff
@@ -16,6 +16,67 @@ import scala.concurrent.{ExecutionContext, Future, Promise}
 /** Useful utilities to use in the wallet project for syncing things against bitcoind */
 object BitcoindRpcBackendUtil extends BitcoinSLogger {
 
+  /** Has the wallet process all the blocks it has not seen up until bitcoind's chain tip */
+  def catchupWalletToBitcoind(bitcoind: BitcoindRpcClient, wallet: Wallet)(
```

Comment:
nit: We use the terminology `sync` elsewhere, if you don't have a strong preference I would prefer to try and be consistent: `syncWalletToBitcoind()`